src/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -1197,6 +1225,7 @@ public abstract partial class XmlWriter : System.IDisposable
         public virtual System.Threading.Tasks.Task FlushAsync() { throw null; }
         public abstract string LookupPrefix(string ns);
         public virtual void WriteAttributes(System.Xml.XmlReader reader, bool defattr) { }
+        [System.Diagnostics.DebuggerStepThroughAttribute]
```

Comment:
Are these attributes required in the ref? If so, can we fix this in the tool? cc @ericstj

Reply:
https://github.com/dotnet/corefx/blob/master/eng/DefaultGenApiDocIds.txt